website/ref/language.md

```diff
@@ -2432,6 +2432,32 @@ use a/b/c # imports the "a/b/c" module as "c:"
 use a/b/c foo # imports the "a/b/c" module as "foo:"
 ```
 
+### Conditionally importing modules with `use`
```

Comment:
This example already exists in https://elv.sh/ref/edit.html#edit:add-var.

It also doesn't belong to the reference doc - it's not part of the language, and the entire section is non-normative.

It's better to expand the tip in https://elv.sh/ref/edit.html#edit:add-var a bit and add a link here.